lib/blockscore/base.rb

```diff
@@ -31,6 +49,7 @@ def save
 
     def save!
```

Comment:
While looking at this I noticed the other save method has a bare rescue which means it just swallows up ALL errors.  I thought I caught this back when the gem was first released but I guess not.  Would you mind fixing that and double checking this isn't done elsewhere in the lib?